### DISCUSSION TITLE
UploadFile not upload the file and no errors occurs

### DISCUSSION THREAD

**Discussion by diegocurbelo12** [sshnet/SSH.NET#1463] [Q&A]
Hi, I have a weird error.
The "UploadFile" operation does not upload a file and does not return any errors. I can connect to the sftp server, I can list files with the library but when I try to upload a file, all code run but no exception or error occurs and the file is not there.
Using FileZilla, I was able to upload files without any issues.
What can I do?

**Comment by Rob-Hague** [comment]
Not got an easy answer, I suppose you could either step through the library code (sourcelink is enabled), or try passing a value for the `uploadCallback` argument and printing some debug information from there to see where the code is getting to
